src/internal/m365/service/exchange/export_test.go

```diff
@@ -0,0 +1,275 @@
+package exchange
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/data"
+	dataMock "github.com/alcionai/corso/src/internal/data/mock"
+	"github.com/alcionai/corso/src/internal/m365/collection/exchange"
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/internal/version"
+	"github.com/alcionai/corso/src/pkg/control"
+	"github.com/alcionai/corso/src/pkg/export"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+const testEmailPath = "../../../converters/eml/testdata/email-with-attachments.json"
```

Comment:
The json file was pretty big and I thought it might be easily to navigate or make edits in a json file when needed instead of as as a long unrecognized string in a go file. I could maybe look into using [embed](https://pkg.go.dev/embed) if you just need the semantics. Let me know if you have any strong opinions on it and I can move the string into a go file.

Reply:
I am not a big fan of specifying the path like this especially as this is prone to breaking if we move around the file. Embed might be an alternative for that.

Reply:
yea, I'd be more comfortable without the explicit path as well. If there's an easy way to avoid having to specify it that'd be a great change to make